ODLA/platforms/tensorrt/plugins/batchedNMSPlugin/batchedNMSInference.cpp

```diff
@@ -122,28 +122,50 @@ pluginStatus_t nmsInference(
 
   void* sortingWorkspace =
       nextWorkspacePtr((int8_t*)postNMSIndices, postNMSIndicesSize);
-  // Sort the scores so that the following NMS could be applied.
+// Sort the scores so that the following NMS could be applied.
+#if NV_TENSORRT_MAJOR >= 8
+  int score_bits = 1;
```

Comment:
Based on TRT8 BatchedNMSPlugin/README,  BatchedNMSPluginCreator::createPlugin, the default value of score_bits should be 16. It works for DT_HALF as 
    float scoreShift = 0.f;
    if(DT_SCORE == DataType::kHALF && scoreBits > 0 && scoreBits <= 10)
        scoreShift = 1.f;   
So score_shift = 0 is a correct default value. 
In summary, better to set score_bits = 16.

Reply:
Thanks